project/build.sbt

```diff
@@ -3,7 +3,7 @@ libraryDependencies ++= Seq(
   "org.sonarsource.update-center" % "sonar-update-center-common" % "1.21.0.561",
   // Scapegoat & scalastyle inspections generator dependencies
   "com.sksamuel.scapegoat" %% "scalac-scapegoat-plugin" % "1.3.7",
-  "io.github.lukehutch" % "fast-classpath-scanner" % "3.1.13",
+  "io.github.classgraph" % "classgraph" % "4.1.6",
```

Comment:
👍

Reply:
You'll need to bump this version to classgraph/classgraph-4.1.7 if there's a chance you will be running in a modular (JPMS) classpath -- I just pushed out an important fix yesterday for this case.

Reply:
Hi @lukehutch, thanks for the ping, we'll make sure to bump it before the next release :+1: